SPECS/qt5-qtbase/qt5-qtbase.spec

```diff
@@ -148,6 +148,11 @@ Patch87: CVE-2023-38197.patch
 # Fix CVE-2023-51714
 Patch88: CVE-2023-51714.patch
 
+# Backport: https://github.com/qt/qtbase/commit/6869d2463a2e0d71bd04dbc82f5d6ef4933dc510
+Patch89: avoid-processing-intensive-painting-of-tiny-dashes.patch
```

Comment:
Why was this patch needed to patch the CVE? Could you extend the comment, so that we know why we need this patch?